main.java

```diff
@@ -1,9 +1,18 @@
 class Main{
     
     public static void main(String[] args) {
+        System.out.println("Give attack for Knigh1");
+        int Knight1ATK=ReadLine();
+        System.out.println("Now hp");
+        int Knight1HP=ReadLine();
 
-        Unit Knight1 = new Unit(5,10);
-        Unit Knight2 = new Unit(6,10);
+        System.out.println("Give attack for Knigh2");
+        int Knight2ATK=ReadLine();
+        System.out.println("Now hp");
+        int Knight2HP=ReadLine();
+
+        Unit Knight1 = new Unit(Knight1ATK,Knight1HP);
+        Unit Knight2 = new Unit(Knight2ATK,Knight2HP);
 
         battle(Knight1, Knight2);
```

Comment:
Nem az uj reszben van, de a battle-ben  a Knight2 akkor is visszaut, ha mar meghalt.